src/native/containers/dn-umap.c

```diff
@@ -144,13 +144,12 @@ static void
 umap_rehash (dn_umap_t *map)
 {
 	uint32_t diff;
-
 	if (map->_internal._last_rehash > map->_internal._node_count)
 		diff = map->_internal._last_rehash - map->_internal._node_count;
 	else
 		diff = map->_internal._node_count - map->_internal._last_rehash;
 
-	if (!((double)diff * 0.75 > (double)map->_internal._bucket_count * 2))
+	if (!(diff * 0.75 > map->_internal._bucket_count * 2))
```

Comment:
```suggestion
	if (!(diff * 0.75 > map->_internal._bucket_count * 2.0))
```

Reply:
OK, copy paste from ghashtable.c, so guess it should still be OK.